jquery.dynatable.js

```diff
@@ -114,6 +114,23 @@
       sortTypes: {},
       records: null
     },
+    ajax: {
+      ajaxComplete: function(response) {
+        this.$element.trigger('dynatable:ajax:success', response);
+        // Merge ajax results and meta-data into dynatables cached data
+        this.records.updateFromJson(response);
+        // update table with new records
+        this.dom.update();
+
+        if (obj.skipPushState && obj.state.initOnLoad()) {
```

Comment:
I'd probably keep the way we reference the `this` object consistent within the function here; seems like we randomly switch from directly calling `this` to then start calling the equivalent `obj`. I'd do it one way or the other.